tests/console/scc_cleanup_reregister.pm

```diff
@@ -26,6 +27,7 @@ sub run {
     register_product;
     my $addons = get_test_suite_data()->{addons};
     register_addons_cmd($addons);
+    deregister_dropped_modules if (get_var('DROPPED_MODULES') || get_var('SCC_ADDONS', '') =~ /ltss/);
```

Comment:
you don't need the condition, it is already inside the function.

Reply:
Removed the condition and update the VRs.